### PR TITLE
clarify SetSubnetValidatorManagerTx

### PR DESCRIPTION
**Pull request by ceyonur**
It's a bit hard to follow the difference between `SourceAddress` and `Address`. We should clarify that these are two different fields in the warp message with `AddressedCall` payload

### CODE REVIEW

File: ACPs/77-reinventing-subnets/README.md [hunk 1]
```diff
@@ -192,7 +194,8 @@ type SetSubnetValidatorManagerTx struct {
     // Warp message should include:
     //   - SubnetID
     //   - ChainID (where the validator manager lives)
```
Comment (ceyonur) [inline ACPs/77-reinventing-subnets/README.md:196]: Is this same with `SourceChainID` or is that also a separate field in `AddressedCall`? cc @dhrubabasu

**Comment by dhrubabasu** [comment]
All the `// Warp message should include:` comments are the payload for the `AddressedCall`. The `SourceAddress` isn't listed there. Definitely could be clearer, thanks for looking at it!

**Comment by ceyonur** [comment]
We've removed `SetSubnetValidatorManagerTx` and fixed those in https://github.com/avalanche-foundation/ACPs/pull/135
